pro/src/screens/IndividualOffer/UsefulInformationScreen/utils.ts

```diff
@@ -49,6 +54,37 @@ export function setDefaultInitialValuesFromOffer(
     (value) => value === false
   )
 
+  let addressFields = {}
+  if (offer.address) {
+    const { street, postalCode, city, latitude, longitude } = offer.address
+    const addressAutocomplete = `${street} ${postalCode} ${city}`
```

Comment:
Histoire que les changements d'affichage soient répércutés partout
```suggestion
    const addressAutocomplete = computeAddressDisplayName(offer.address, false)
```